### PR TITLE
Bugfix/jinja2 change delimiters

### PR DESCRIPTION
**Pull request by liortct**
Addresses - #35 
This PR has been ported over from the Cookiecutter project.
For more details see: https://github.com/cookiecutter/cookiecutter/pull/1767

### CODE REVIEW

File: cookieninja/generate.py [hunk 1]
```diff
@@ -291,15 +294,15 @@ def generate_files(
     :param keep_project_on_failure: If `True` keep generated project directory even when
         generation fails
     """
-    template_dir = find_template(repo_dir)
+    envvars = context.get('cookiecutter', {}).get('_jinja2_env_vars', {})
```
Comment (yuvaldolev) [inline cookieninja/generate.py]:
```suggestion
    jinja2_env_vars = context.get('cookiecutter', {}).get('_jinja2_env_vars', {})
```

File: docs/advanced/jinja2_custom_delimiter.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.. _jinja2-custom-delimiter:
+
+Jinja2 custom delimiter
+-----------------------
+
+You can specify an ``_jinja2_env_vars`` in ``cookiecutter.json`` file for change the default jinja2 delimiter ``{{``.
```
Comment (yuvaldolev) [inline docs/advanced/jinja2_custom_delimiter.rst]:
```suggestion
You can specify a dictionary under the ``_jinja2_env_vars`` key in the ``cookiecutter.json`` file to modify the default jinja2 delimiter (``{{``).
```

File: docs/advanced/jinja2_custom_delimiter.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+This is very useful when you templates contains files like:
```
Comment (yuvaldolev) [inline docs/advanced/jinja2_custom_delimiter.rst]:
```suggestion
This is very useful for templates which contain files that have expressions with that use the default delimiters ``{{ ... }}``  such as:
```

File: docs/advanced/jinja2_custom_delimiter.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+1. Helm Chart
```
Comment (yuvaldolev) [inline docs/advanced/jinja2_custom_delimiter.rst]:
```suggestion
1. Helm Charts
```

**Review by yuvaldolev — CHANGES_REQUESTED** [review]
Very nice :)

File: cookieninja/generate.py (hunk 1 above)
Comment (liortct) [reply cookieninja/generate.py]: Fixed

File: docs/advanced/jinja2_custom_delimiter.rst (hunk 2 above)
Comment (liortct) [reply docs/advanced/jinja2_custom_delimiter.rst]: Fixed

File: docs/advanced/jinja2_custom_delimiter.rst (hunk 3 above)
Comment (liortct) [reply docs/advanced/jinja2_custom_delimiter.rst]: Fixed

File: docs/advanced/jinja2_custom_delimiter.rst (hunk 4 above)
Comment (liortct) [reply docs/advanced/jinja2_custom_delimiter.rst]: Fixed

File: docs/advanced/jinja2_custom_delimiter.rst [hunk 5]
```diff
@@ -3,10 +3,10 @@
 Jinja2 custom delimiter
 -----------------------
 
-You can specify an ``_jinja2_env_vars`` in ``cookiecutter.json`` file for change the default jinja2 delimiter ``{{``.
-This is very useful when you templates contains files like:
+You can specify a dictionary under the ``_jinja2_env_vars`` key in the ``cookiecutter.json`` file to modify the default jinja2 delimiter (``{{``).
+This is very useful for templates which contain files that have expressions with that use the default delimiters ``{{ ... }}``  such as:
```
Comment (yuvaldolev) [inline docs/advanced/jinja2_custom_delimiter.rst]:
```suggestion
This is very useful for templates which contain files that have expressions that use the default delimiters ``{{ ... }}``  such as:
```

**Review by yuvaldolev — CHANGES_REQUESTED** [review]

**Review by yuvaldolev — APPROVED** [review]
Very nice!

**Pull request merged**